src/test/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkTaskWithSchemaTest.java

```diff
@@ -32,7 +32,7 @@ public class ClickHouseSinkTaskWithSchemaTest {
     private static ClickHouseHelperClient chc = null;
 
     @BeforeAll
-    private static void setup() {
+    public static void setup() {
```

Comment:
same here
public can be also removed according https://www.baeldung.com/junit-before-beforeclass-beforeeach-beforeall